Fix various typos.

roberthoenig opened the pull request.

MrDupin commented:
Thanks for the help, nice catches!

A little request: Please thin out PRs that edit a lot of notebooks. When this gets merged, it's very likely that merge conflicts will arise. Merge conflicts on notebooks are not that simply to fix so it will require a lot of work from the rest.

Thanks!

roberthoenig commented:
Thanks for the review! Re merge conflicts: Sounds reasonable, I’ll watch out for notebooks in the future.  What’s the next step for this PR to get merged? The options I see are 
* leave it as it is.
* split into two commits, one for non-notebook and one for notebook, so the first one can easily get cherry picked.
* split into two PRs, so the first one can easily get merged.

Sent from my iPhone

> On Feb 16, 2018, at 8:56 PM, Anthony Marakis <notifications@github.com> wrote:
> 
> Thanks for the help, nice catches!
> 
> A little request: Please thin out PRs that edit a lot of notebooks. When this gets merged, it's very likely that merge conflicts will arise. Merge conflicts on notebooks are not that simply to fix so it will require a lot of work from the rest.
> 
> Thanks!
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

MrDupin commented:
It is up to you, but I would leave it as is. If in the future you see a PR already editing a notebook with a typo, you can message the PR creator and let them know of the typo so that they can fix it.
